mne/decoding/base.py

```diff
@@ -364,46 +375,77 @@ class LinearModel(MetaEstimatorMixin, BaseEstimator):
     .. footbibliography::
     """
 
-    # TODO: Properly refactor this using
-    # https://github.com/scikit-learn/scikit-learn/issues/30237#issuecomment-2465572885
     _model_attr_wrap = (
         "transform",
+        "fit_transform",
         "predict",
         "predict_proba",
-        "_estimator_type",
-        "__tags__",
+        "predict_log_proba",
+        "_estimator_type",  # remove after sklearn 1.6
         "decision_function",
         "score",
         "classes_",
     )
 
     def __init__(self, model=None):
-        # TODO: We need to set this to get our tag checking to work properly
+        # XXX Remove the clause after warning cycle
         if model is None:
             model = LogisticRegression(solver="liblinear")
+            depr_message = (
+                "Starting with mne-python v1.13 'model' default "
+                "will change from LogisticRegression to None. "
+                "From now on please set model=LogisticRegression"
+                "(solver='liblinear') explicitly."
+            )
+            warn(depr_message, FutureWarning)
+
```

Comment:
So instead of this, we just remove this `if model is None` check entirely into the `fit` step